sdk/keyvault/azure-security-keyvault-keys/src/cryptography/rsa_cryptography_provider.cpp

```diff
@@ -53,4 +53,28 @@ namespace Azure {
       return result;
     }
 
+    SignResult RsaCryptographyProvider::Sign(
+        SignatureAlgorithm const& algorithm,
+        std::vector<uint8_t> const& digest,
+        Azure::Core::Context const&) const
+    {
+      SignResult result;
```

Comment:
Is implementation coming later? For now, you should throw something to say "not implemented" effectively.

Reply:
Yes, I added this just as the model on how a local provider would be there to perform crypto locally and if it throw or let fields empty, we use remote client.  So, yes, either I can throw or do nothing for now until I add the impl for openSSL (linux) and bcrypt.h (Win)